device.mk

```diff
@@ -205,4 +205,26 @@ PRODUCT_PROPERTY_OVERRIDES += \
 #
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
     persist.sys.usb.config=mtp \
```

Comment:
We need to update it to  mass_storage, but update to mass_storage may cause usb can't be detected by host. I think this won't block our development of aries-l and take it as a different issue and fix it when there is a solution.